synapse/config/_base.py

```diff
@@ -123,7 +123,9 @@ def format_config_error(e: ConfigError) -> Iterator[str]:
 # should have the same indentation.
 #
 # [1] https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html
-
+#
+# For more information on how to configure Synapse, including a complete accounting of 
+# each option, go to docs/usage/configuration/config_documentation.md
```

Comment:
Should this point to the config manual URL? I think this is going to be written to generated config files, see `synapse.config._base.RootConfig.generate_config`.

Reply:
The idea was that users would be pointed to the documentation that is available offline, I can add a link to the URL as well.

Reply:
Ahh, I see. Both would be nice IMO, but don't let that block this.